.travis.yml

```diff
@@ -12,13 +12,25 @@ cache:
   directories:
       - $HOME/cachedir
 
+services:
```

Comment:
Don't oppose but isn't always included?

https://docs.travis-ci.com/user/database-setup/#mysql

Reply:
Not in Xenial https://docs.travis-ci.com/user/reference/xenial/#services-disabled-by-default

Reply:
yeah, had to add it recently to a number of plugins too.